index.bs

```diff
@@ -1110,6 +1110,16 @@ An <dfn>eligibility</dfn> is one of the following:
 
 </dl>
 
+A <dfn>registrar</dfn> is one of the following:
+
+<dl dfn-for="registrar">
```

Comment:
Can we remove the `support key` definition below if we're going to add this definition? They are identical.